burn-tensor/src/tensor/api/numeric.rs

```diff
@@ -1589,6 +1589,50 @@ where
     /// For calculating abs of the elements of a tensor, users should prefer the [Tensor::abs](Tensor::abs) function,
     /// which is more high-level and designed for public use.
     fn abs<const D: usize>(tensor: Self::Primitive<D>) -> Self::Primitive<D>;
+
```

Comment:
In this file, you should actually add the user API operation implementations where the static dispatch is done. This isn't inside a trait, but the implementation uses the Kind to dispatch to the right implementation.

Example with powf_scalar:

```rust
    /// Applies element wise powf operation with a scalar.
    ///
    /// `y = x + s`
    pub fn powf_scalar<E: ElementConversion>(self, other: E) -> Self {
        Self::new(K::powf_scalar(self.primitive, other))
    }
```